charts/k8ssandra/crds/medusa/backup.yaml

```diff
@@ -3817,14 +3722,7 @@ spec:
                       type: string
                     serverVersion:
                       description: Version string for config builder, used to generate Cassandra server configuration
-                      enum:
-                        - 6.8.0
-                        - 6.8.1
-                        - 6.8.2
-                        - 6.8.3
-                        - 3.11.6
-                        - 3.11.7
-                        - 4.0.0
+                      pattern: (6\.8\.\d+)|(3\.11\.\d+)|(4\.0\.\d+)
```

Comment:
This is more of a question, but will `4.0-beta4` match `4.0`? The pattern will only handle digits it seems, so I guess I'm not sure where the version here maps to an image tag.

Reply:
That is a great question. It looks like the answer is no. We will have to open an issue in cass-operator. I can do that.

Reply:
will do

Reply:
done: https://github.com/datastax/cass-operator/issues/373